CHANGELOG.md

```diff
@@ -1,5 +1,63 @@
 # Changelog
 
+## [1.0.0] - 2021-07-08
+
+### Added
+- The `vault` UpstreamAuthority plugin now supports Kubernetes service account authentication (#2356)
+- A new `cert-manager` UpstreamAuthority plugin is now available (#2274)
+- SPIRE Server CLI can now be used to ban agents (#2374)
+- SPIRE Server CLI now has `count` subcommands for agents, entries, and bundles (#2128)
+- SPIRE Server can now be configured for SPIFFE federation using the `https_web` and `https_spiffe` profiles (#2340)
+- SPIRE Server and Agent now expose the standard gRPC health service (#2057, #2058)
+- SPIFFE bundle endpoint URL is now configurable (#2278)
```

Comment:
#2278 is the issue, the PR is #2340. I think that it would be good to make it clear that this is referring to the foreign trust domains when configuring federation relationships, not the SPIFFE bundle endpoint that may be served by _this_ SPIRE Server.